content/docs/pipelines/sdk/sdk-overview.md

```diff
@@ -117,6 +117,34 @@ The Kubeflow Pipelines SDK includes the following packages:
   Kubeflow Pipelines. Examples include utility functions for on premises,
   Google Cloud Platform (GCP), Amazon Web Services (AWS), and Microsoft Azure.
 
+* [KFP diagnose_me modules](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp/cli/diagnose_me)
+  These modules are part of the KFP CLI tool that are designed to help with environment diagnostic tasks such as 
+  ensuring required kubernetes secrets are set properly. 
+ 
+  * `kfp.cli.diagnose_me.dev_env` collects developer env related environment configurations such as python version.
+  * `kfp.cli.diagnose_me.kubernetes_cluster` collects Kubernetes cluster related configurations such as secrets.
+  * `kfp.cli.diagnose_me.gcp` collects Google Cloud Platform related environment configurations.
+ 
+## KFP CLI Tool 
+Kubeflow pipelines has a CLI tool that enables a subset of the sdk functionality directly from a command line in addition to environment diagnostic features. In particular the CLI tool provides the following features: 
+
+* `kfp diagnose_me` runs environment diagnostic with specified parameters.
+  * `--json` output in Json format, human readable format is set by default.
+  * `--project-id TEXT` target project id. It will use environment default if not specified.
+  * `--namespace TEXT`   Namespace to use for Kubernetes cluster.all-namespaces is.
+  
+* `kfp pipeline Command` manages pipeline resources.
```

Comment:
Suggested revisions:

* `kfp pipeline <COMMAND>` provides the following commands to help you manage pipelines.
  * `get` - Gets detailed information about a Kubeflow pipeline from your Kubeflow Pipelines cluster.
  * `list` - Lists the pipelines that have been uploaded to your Kubeflow Pipelines cluster.
  * `upload` - Uploads a pipeline to your Kubeflow Pipelines cluster.

Reply:
Done